MekHQ/src/mekhq/service/MassRepairService.java

```diff
@@ -805,11 +805,7 @@ private static MassRepairPartAction repairPart(CampaignGUI campaignGUI, IPartWor
 					continue;
 				}
 
-				if (configuredOptions.isOptimizeToCompleteToday()) {
-					isSameDayTech = false;
-				} else {
-					isSameDayTech = true;
-				}
+                isSameDayTech = !configuredOptions.isOptimizeToCompleteToday();
```

Comment:
Looks like mixed tabs/spaces in this file

Reply:
Yep, pretty much the whole file was tab spaced. Fixed